src/main/java/th/in/gamer/controller/UserController.java

```diff
@@ -0,0 +1,53 @@
+package th.in.gamer.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import th.in.gamer.beans.User;
+import th.in.gamer.user.response.UserStatusResponse;
+import th.in.gamer.user.services.UserService;
+
+@RestController
+@RequestMapping("/api/v1")
```

Comment:
don't use `/api/v1`
normally `api` keyword should located in host name like `api.gamer.in.th`
and `/v1` should be handled by gateway.